### PR TITLE
LATAM : Adiciones múltiples (una vez más)

### PR DESCRIPTION
**Pull request by BellezaEmporium**

### CODE REVIEW

**Review by its0x08 — DISMISSED** [review]

File: streams/us.m3u [hunk 1]
```diff
@@ -213,8 +213,8 @@ https://cdn.herringnetwork.com/80A4DFF/awee_nva/AWE_Encore.m3u8
 https://www.livedoomovie.com/02_AXNHD_720p/chunklist.m3u8
 #EXTINF:-1 tvg-id="AXNEastAsia.us" status="error",AXN East Asia (Vietnamese) (1080p)
 http://103.81.104.118/hls/stream2.m3u8
-#EXTINF:-1 tvg-id="AXNLatinoamerica.us" status="timeout",AXN Latinoamérica (576p)
-http://209.91.213.10:8088/play/a011
+#EXTINF:-1 tvg-id="AXNLatinoamerica.us",AXN Latinoamérica (1080p)
+http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.twitch.tv/anxcx96
```
Comment (freearhey) [inline streams/us.m3u:217]: image

**Review by freearhey — CHANGES_REQUESTED** [review]
Please check the marked link again.

**Comment by LaneSh4d0w** [comment]
> Please check the marked link again.

Going to check in a few hours, not available for now.

File: streams/mx.m3u [hunk 2]
```diff
@@ -29,8 +29,8 @@ https://5e50264bd6766.streamlock.net/canal442/videocanal442/playlist.m3u8
 https://5e50264bd6766.streamlock.net/canal44/videocanal44/playlist.m3u8
 #EXTINF:-1 tvg-id="CanalCatorce.mx" status="online",Canal Catorce (720p)
 https://d3nljkrx6mjqra.cloudfront.net/out/v1/1b9d9efd27814b3b8dc570113ae54409/index.m3u8
-#EXTINF:-1 tvg-id="CanaldelasEstrellas.mx" status="error",Canal de Las Estrellas (480p)
-http://170.83.242.153:8000/play/a012
+#EXTINF:-1 tvg-id="CanaldelasEstrellas.mx",Canal de Las Estrellas (480p)
+http://edge03-fdo-py.cvattv.com.ar/live/c6eds/Canal_de_las_estrellas/SA_SAGEMCOM/Canal_de_las_estrellas.m3u8
```
Comment (freearhey) [inline streams/mx.m3u:33]: image

File: streams/us.m3u [hunk 3]
```diff
@@ -213,8 +213,8 @@ https://cdn.herringnetwork.com/80A4DFF/awee_nva/AWE_Encore.m3u8
 https://www.livedoomovie.com/02_AXNHD_720p/chunklist.m3u8
 #EXTINF:-1 tvg-id="AXNEastAsia.us" status="error",AXN East Asia (Vietnamese) (1080p)
 http://103.81.104.118/hls/stream2.m3u8
-#EXTINF:-1 tvg-id="AXNLatinoamerica.us" status="timeout",AXN Latinoamérica (576p)
-http://209.91.213.10:8088/play/a011
+#EXTINF:-1 tvg-id="AXNLatinoamerica.us",AXN Latinoamérica (1080p)
+http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.twitch.tv/anxcx913
```
Comment (freearhey) [inline streams/us.m3u:217]: works fine (via VPN in my case)

File: streams/us_glewedtv.m3u [hunk 4]
```diff
@@ -1,3 +1,5 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="TheFirst.us" status="online",The First (1080p) [Not 24/7]
 https://redseat-thefirst-glewedtv.amagi.tv/index.m3u8
+#EXTINF:-1 tvg-id="EstrellaTV.mx" status="online",Estrella TV (1080p)
+https://estrellatv-glewed.amagi.tv/playlist.m3u8
```
Comment (freearhey) [inline streams/us_glewedtv.m3u:5]: also works

**Review by freearhey — CHANGES_REQUESTED** [review]
the problem is only with the first link

File: streams/mx.m3u (hunk 2 above)
Comment (LaneSh4d0w) [reply streams/mx.m3u:33]: Works in my case, tested with VLC.
vlcsnap-2022-03-23-22h05m20s206

**Review by freearhey — DISMISSED** [review]

**Comment by sguinetti** [comment]
@LaneSh4d0w I fix the title, is "múltiples" with tittle, similar with "más"

**Comment by LaneSh4d0w** [comment]
> @LaneSh4d0w I fix the title, is "múltiples" with tittle, similar with "más"

Thanks, i do not have these accents in my keyboard.

**Comment by freearhey** [comment]
@LaneSh4d0w Are those links still working for you?

image

**Comment by LaneSh4d0w** [comment]
> @LaneSh4d0w Are those links still working for you?
> 
> image

I'll remove the "stream2" links, it gives me a 403. The others still work.

**Comment by LaneSh4d0w** [comment]
Guess the Argentinian links stopped too.

**Review by freearhey — APPROVED** [review]

**Review by sguinetti — APPROVED** [review]

**Pull request merged**